picard/formats/__init__.py

```diff
@@ -10,6 +10,7 @@
 # Copyright (C) 2016-2018 Sambhav Kothari
 # Copyright (C) 2017 Sophist-UK
 # Copyright (C) 2017 Ville Skyttä
+# Copyright (C) 2020 Gabriel Ferreira
```

Comment:
This can be removed, as there are no changes in the file.

Reply:
Forgot in the previous PR, but can be removed.